CMakeLists.txt

```diff
@@ -37,8 +37,8 @@ set( SPRING_UTIL_EXECUTABLE_NAME spring-util )
 
 # http://stackoverflow.com/a/18369825
 if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 10.2)
-        message(FATAL_ERROR "GCC version must be at least 10.2.0!")
+    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 11.0)
+        message(FATAL_ERROR "GCC version must be at least 11.0.0!")
```

Comment:
I think we should require 11.4 which is what is on ubuntu22 and is checked on ci/cd.